tests/api/test_datamap.py

```diff
@@ -0,0 +1,36 @@
+# pylint: disable=missing-docstring, redefined-outer-name
+from typing import Generator
+
+import pytest
+from starlette.testclient import TestClient
+
+from fidesapi import main
+from fidesapi.routes.util import API_PREFIX
+from fidesctl.core.config import FidesctlConfig
+
+
+@pytest.fixture()
+def test_client() -> Generator:
```

Comment:
I recommend moving this to `conftest.py`, since it can be used anywhere we need a test client (I think we're also using it elsewhere already too)

Reply:
Agreed 👍🏽  I was dabbling with that in trying to solve this other issue as well. Will add an overall cleanup to the follow on issue I am going to create prior to merging this